_editorial_policies/02_editor_instructions.md

```diff
@@ -35,7 +35,7 @@ Before assigning manuscripts for review, editors have several main tasks:
 ## Review handling
 
 Once an editor has handled the pre-review steps described above, the review process is largely similar to typical journals. The editor:
-- Contacts suitable reviewers to request reviews, giving them a reasonable (typically three weeks) but not extended amount of time to provide their reviews and noting our conflict of interest policy
+- Contacts suitable reviewers to request reviews, giving them a reasonable (typically 10 days, but exceptions can be made) but not extended amount of time to provide their reviews and noting our conflict of interest policy
```

Comment:
so we could change the language to
"Contacts suitable reviewers to request reviews, noting our conflict of interest policy.  Timing and policy details will be automatically provided to reviewers via the form-letter requests.  Reviews will be due in 14 days, though requests for extension will be routinely granted, for an additional 7-10 days."

We could further add, "In the event a reviewer misses an extended deadline (24 days from accepting the review assignment), the editor should request an additional reviewer.  The editor should start the process of conveying reviews to authors once two have been received, unless additional expertise requires waiting for a third review.  In addition to the two peer reviews, a student review must also be obtained.  A student reivew may be edited to ensure anonymity and proper tone, but not for substance."

Reply:
Sounds good to me. Want to change? (PR to my PR?) I'm mostly offline today.